backend/backend.go

```diff
@@ -186,6 +186,21 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 	backend.ratesUpdater = rates.NewRateUpdater(hclient)
 	backend.ratesUpdater.Observe(backend.Notify)
 	backend.ratesUpdater.Start()
+	for _, fiat := range config.AppConfig().Backend.FiatList {
```

Comment:
Mabye this code should be in `initAccounts()`? Then it will work even without an app restart if you toggle accounts in the settings, as toggling accounts will invoke `initAccounts()`.

Just need to take care to that EnableHistoryPair should only run once per pair.

Reply:
Hmm same for when changing the fiat settings. Let's postpone fixing both of it to a new PR.